### DISCUSSION TITLE
CSS Problem after run build

### DISCUSSION THREAD

**Discussion by andrei-bancos** [vercel/next.js#24744] [Help]
### What version of Next.js are you using?

10.2.0

### What version of Node.js are you using?

14.15.5

### What browser are you using?

Chrome, Firefox and Opera

### What operating system are you using?

Linux & Windows

### How are you deploying your application?

Express server

### Describe the Bug

Hello,
I have a problem with my css in production mode.
In dev mode css display correct on the page, but after npm run build & npm start - css is a wrong displayed.
I use sass and module for my css code.

### Expected Behavior

The page layout should be the same in dev and build mode, but is different.

### To Reproduce

![import](https://user-images.githubusercontent.com/42214745/116784755-9f5a9300-aa9e-11eb-9c87-0e953f0aee0a.png)

Before run build (dev mode)
![dev-mode](https://user-images.githubusercontent.com/42214745/116784779-b8fbda80-aa9e-11eb-8234-3db2ebdb1136.png)

After run build (production)
![build](https://user-images.githubusercontent.com/42214745/116784805-daf55d00-aa9e-11eb-88f6-089ec2196473.png)

**Comment by krychaxp** [comment]
to low information, show your code
